__docs__/wonder-blocks-popover/popover.stories.tsx

```diff
@@ -338,6 +343,23 @@ WithInitialFocusId.parameters = {
     },
 };
 
+/**
+ * Popovers can have custom layouts. This is done by using the
+ * `PopoverContentCore` component.
+ *
+ * _NOTE:_ If you choose to use this component, you'll have to set the
+ * `aria-labelledby` and `aria-describedby` attributes manually. Make sure to
+ * pass the `id` prop to the `Popover` component and use it as the value for
+ * these attributes. Also, make sure to assign the `${id}-title` prop to the
+ * `title` element and `${id}-content` prop to the `content` element.
+ */
+export const CustomPopoverContent: StoryComponentType = {
```

Comment:
**note:** I noticed that we didn't include a story with custom popovers, so I added it here. This also helps to manually test custom popovers with screen readers.